MonoGame.Framework/Graphics/Texture2D.DirectX.cs

```diff
@@ -30,19 +30,14 @@ namespace Microsoft.Xna.Framework.Graphics
 {
     public partial class Texture2D : Texture
     {
-        private bool _shared;
-
-        private bool _renderTarget;
-        private bool _mipmap;
-
-        private SampleDescription _sampleDescription;
+        protected internal bool shared;
+        protected internal bool mipmap;
```

Comment:
I'd be fine with those fields staying private and with the underscore prefix (as they were before this PR), and a adding new protected PascalCase property used by derived classes that returns the private field.